assets/sass/components/global/_googlesitekit-new-badge.scss

```diff
@@ -19,8 +19,11 @@
 .googlesitekit-new-badge {
 	background-color: $c-violet-v-500;
 	color: $c-violet-v-0;
+	font-size: $fs-title-sm;
+	line-height: $lh-title-sm;
```

Comment:
Looking at our type scale, this should use what we have defined for `12/16` under the Label group which uses `$fs-display-sm` and `$lh-display-sm` respectively.

See https://google.github.io/site-kit-wp/storybook/pull/6805/?path=/story/global-typescale--default-type-scale

Reply:
I have updated them to `$fs-label-sm` and `$lh-label-sm` assuming that's what you meant.  :)

Reply:
Ah yes, thanks 👍